django_project/base/models/project.py

```diff
@@ -158,6 +158,13 @@ class Project(models.Model):
         null=True
     )
 
+    project_repository_url = models.URLField(
+        help_text=u'A repository url for this project '
+                  u'e.g a github repository url.',
```

Comment:
Please check carefully my comment:

```
      help_text=u'A repository url for this project '
                 u'e.g a github repository url.',
```

should be

```
      help_text=u'A repository URL for this project '
                 u'e.g, a GitHub repository URL.',
```

Also your string contruction is strange, it should be:

```
      help_text=(
          u'A repository url for this project '
          'e.g. a GitHub repository URL.'),
```